lib/modules/versioning/ubuntu/ubuntu-distro-json-generate.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
```

Comment:
Copied and adjusted the comment from @viceice so it can be tracked and resolved later. 😉 

> Would like to have it as `.mjs` file, so it works on any platform, like our other scripts in the `tools/` folder. That's also the right place for it.

Reply:
mjs file added 😄